pkg/sql/sqlbase/structured.go

```diff
@@ -4279,3 +4279,30 @@ func (desc ColumnDescriptor) GetLogicalColumnID() ColumnID {
 
 	return desc.ID
 }
+
+// GetTypeModifier returns the type modifier of the ColumnDescriptor. This
+// corresponds to the pg_attribute.atttypmod column. atttypmod records
+// type-specific data supplied at table creation time (for example, the maximum
+// length of a varchar column). The value will be -1 for types that do not need
+// atttypmod.
+func (desc ColumnDescriptor) GetTypeModifier() int32 {
```

Comment:
does this make more sense to just be a method on the type itself?

Reply:
Not sure I follow. Do you mean to make a new typedef for `TypeModifier` or something?

Reply:
I meant that it seems like this should be a function on a types.T, rather than a Column Descriptor because it is only using information present in the type itself.

Reply:
got it, yeah that makes sense. i had misread which function this comment was on.

Reply:
or, maybe I should just make a function in pg_catalog, like what was done for typLen 

```
func typLen(typ *types.T) *tree.DInt {
```

Reply:
nvm, that doesn't work as well, since a lot of places outside need it too